reactioncommerce/reaction-docs
# docs: *remove* Hooks docs and other methods

Resolves #757 
Resolves #760 
Impact: **minor**  
Type: **docs**

## Issue
- Collection hooks, method hooks, event hooks have all been removed in 2.0 (at least). Documentation of these still exists, and some docs use hook methods in examples.
- Other Meteor methods, plugins and helpers are no longer in the code.

## Solution & Screenshots
- Remove docs for Collection, Method and Event hooks
- Update `Hooks.Events` to use `appEvents`
- Update all links to old docs to older versions of those docs
- Add `appEvents` docs
- Remove deprecated autoValue methods
- Add deprecation note for cart/copyCartToOrder, cart/submitPayment, shop/getCurrencyRates, accounts/sendWelcomeEmail

## Breaking changes
None

## Testing
- Make sure all links still work
- Make sure tutorial makes sense
- Make sure docs make sense
